src/extension.ts

```diff
@@ -274,22 +543,53 @@ function updateOcSdkVersion(command: string, ocSdkCmd: OcSdkCommand, session: Se
 }
 
 function initOperatorCollection(command: string, session: Session, outputChannel?: vscode.OutputChannel): vscode.Disposable {
-  return vscode.commands.registerCommand(command, async (logPath?: string) => {
```

Comment:
`logPath` may also be needed here for testing purposes. This is used to log the command output to display for test failures